docs/hazmat/primitives/key-derivation-functions.rst

```diff
@@ -737,6 +737,90 @@ Different KDFs are suitable for different tasks such as:
         The counter iteration variable will be concatenated after
         the fixed input data.
 
+.. currentmodule:: cryptography.hazmat.primitives.kdf.scrypt
+
+.. class:: Scrypt(salt, length, n, r, p, backend)
+
+    .. versionadded:: 1.6
+
+    Scrypt is a KDF designed by Colin Percival to be resistant against
+    hardware-assisted attackers by having a tunable memory cost. It is
+    described in `RFC 7914`_.
```

Comment:
Should probably mention that it is designed for password storage. Also you can use `:rfc:`7914`` to link to an RF